### PR TITLE
README

### PR DESCRIPTION
**Pull request by Casao**
Update README with Partners URL

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-First things first. You're going to need to get a developer key from Careerbuilder. You can obtain a key at http://api.careerbuilder.com/RequestDevKey.aspx.
+First things first. You're going to need to get a developer key from Careerbuilder. To do this, you can reach out to the Partner Team at http://developer.careerbuilder.com/partner_messages/new and request a Developer key.
```
Comment (ausmith) [inline README.md]: I believe it's less "reach out to the partner team" and more "to get a developer key you need to be a partner, to become one, fill out this form".

Comment (cmompoint1) [inline README.md]: +1 to Aaron's comment

File: README.md [hunk 2]
```diff
@@ -1,4 +1,4 @@
-First things first. You're going to need to get a developer key from Careerbuilder. You can obtain a key at http://api.careerbuilder.com/RequestDevKey.aspx.
+First things first. You're going to need to get a developer key from Careerbuilder. Developer keys are available to Partners. You can fill out the form at http://developer.careerbuilder.com/partner_messages/new to become a partner.
```
Comment (Casao) [inline README.md]: @ausmith @cmompoint1 Tweaked the wording, thoughts?

Comment (ausmith) [inline README.md]: :+1: here, @cmompoint1 ?

Comment (tkling) [inline README.md]: Wording looks good to me. Tiny nitpick - 'partner' has inconsistent capitalization.

Comment (cmompoint1) [inline README.md]: I like the wording and let's just capitalize Partner.  Make them feel good :)

**Comment by ausmith** [comment]
Woo! I'm going to not bother asking for a version bump here because that doesn't make any sense anyway.

**Pull request merged**